course_content/notebooks/matplotlib_intro.ipynb

```diff
@@ -81,25 +68,15 @@
-       "text": [
-        "<matplotlib.figure.Figure at 0x43c21d0>"
-       ]
-      }
-     ],
-     "prompt_number": 3
+     "outputs": []
```

Comment:
Did you intend to remove the output here ?
I think that this one is better left in, as a pre-loaded demonstration of the Figure.

I know that saving with output in some cells and the rest clear is pretty awkward, 
but it seems to be part of the original design intent.

In general, if we can control it, then I think leaving specific outputs in can make a notebook much more useful when you don't have a live version (e.g. when viewing in GitHub).
For example, "numpy_intro.ipynb" seems to need the outputs to make it readable; they are used as part of the discussion / explanations.

@corinnebosley @djkirkham **Do we feel this is generally necessary and/or worth it ?**